evap/evaluation/templates/base.html

```diff
@@ -211,5 +211,18 @@
         </script>
 
         {% block additional_javascript %}{% endblock %}
+
+        <script type="text/javascript"> 
+            document.querySelectorAll("[data-earliest-possible-date]").forEach(element => {
+                element.addEventListener("change", () => {
+                    if (element.value < element.dataset.earliestPossibleDate) {
+                        element.setCustomValidity(element.dataset.earliestPossibleDateReason);
+                    } else {
+                        element.setCustomValidity("");
+                    }
+                });
+                element.dispatchEvent(new Event("change"));
+            });
+        </script>
```

Comment:
@niklasmohrin and I talked about putting the check into the base.html to not replicate it for each evaluation and to use it for future date selection modals

Reply:
Yes, I thought that if we need to move it away from the use-site, we might as well decouple the names from the "exam date" concept and make it a generic tool. As such, base.html seemed to be the right place. However, I think it could be in the same javascript block as the `setupForms`, as the two procedures are similar.

I don't think a comment is needed

Reply:
Hmm, ok, fine with me